src/lavinmq/launcher.cr

```diff
@@ -4,16 +4,22 @@ require "systemd"
 require "./reporter"
 require "./server"
 require "./http/http_server"
-require "./log_formatter"
-require "./in_memory_backend"
 require "./data_dir_lock"
+require "./logging"
+
+# Setup from env early so we have a logger
+::Log.setup_from_env(
+  backend: Log::IOBackend.new(
+    formatter: LavinMQ::Logging::StdoutLogFormat))
```

Comment:
means that we get logs in specs, in systemd environments etc with wrong format?

Reply:
Dunno what you mean with wrong format? It's the same formatter as will be used again when the log is configured from config.

I don't recall exactly why I added this, but it's to have a "default logger" configured before we've parse the config.